app/src/main/java/com/architect/coders/mu8/remote/RetrofitManager.kt

```diff
@@ -0,0 +1,24 @@
+package com.architect.coders.mu8.remote
+
+import com.architect.coders.mu8.BuildConfig
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class RetrofitManager {
+
+    companion object {
+        const val BASE_URL = BuildConfig.API_BASE_URL
+    }
```

Comment:
Place the companion object at the end of the class as the [Kotlin style guidelines](https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout) state.